tests/AutolinkerSpec.js

```diff
@@ -332,11 +332,6 @@ describe( "Autolinker", function() {
 				} );
 
 
-				it( 'should not match an address with multiple dots', function() {
-					expect( autolinker.link( 'hello:...world' ) ).toBe( 'hello:...world' );
-					expect( autolinker.link( 'hello:wo.....rld' ) ).toBe( 'hello:wo.....rld' );
```

Comment:
Hey @olafleur. It seems these tests are valid because a domain name cannot have more than one consecutive dot. I would say that we shouldn't remove them, and maybe figure out a way instead to allow multiple dots in the path part of the URL but not the domain part.

Maybe we could look for the first `/`, `?`, or `#` after the scheme, and just make sure that there are no multiple consecutive dots in the domain name? We would link if there are multiple consecutive dots after the first `/`, `?`, or `#`, but not if there are multiple dots before it. Thoughts?